packages/domain-picker/src/domain-picker/index.tsx

```diff
@@ -169,6 +177,11 @@ const DomainPicker: FunctionComponent< Props > = ( {
 		onSetDomainSearch( searchQuery );
 	};
 
+	const showErrorMessage = domainSuggestionState === DataState.Failure;
+	const isDomainSearchEmpty = domainSearch.trim()?.length <= 1;
```

Comment:
If trim works, length will defintely work. You probably need the `?` before the trim call. 
```suggestion
	const isDomainSearchEmpty = domainSearch.trim?.().length <= 1; // or domainSearch.trim?.().length ?? false;
```